domains/noahgao.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "noahziheng",
+        "email": "noahgaocn@outlook.com"
+      },
+
+      "record": {
+        "URL": "https://noahziheng.github.io/#/"
```

Comment:
Do you want the website to be on `noahgao.is-a.dev`? If so, please change the record to `"CNAME": "noahziheng.github.io"`

Reply:
Yep, I want to use `noahgao.is-a.dev`. And the record is set to CNAME now.